opm/simulators/linalg/MILU.cpp

```diff
@@ -232,17 +232,17 @@ void milun_decomposition(const M& A, int n, MILU_VARIANT milu, M& ILU,
         detail::milu0_decomposition ( ILU);
         break;
     case MILU_VARIANT::MILU_2:
-      detail::milu0_decomposition ( ILU, identityFunctor<double>(),
-				    signFunctor<double>()); 
+      detail::milu0_decomposition ( ILU, identityFunctor<double>,
+				    signFunctor<double>); 
      
         break;
     case MILU_VARIANT::MILU_3:
-      detail::milu0_decomposition ( ILU, absFunctor<double>(),
-				    signFunctor<double>());
+      detail::milu0_decomposition ( ILU, absFunctor<double>,
+				    signFunctor<double>);
         break;
     case MILU_VARIANT::MILU_4:
-      detail::milu0_decomposition ( ILU, identityFunctor<double>() ,
-				    isPositiveFunctor<double>());
+      detail::milu0_decomposition ( ILU, identityFunctor<double> ,
+				    isPositiveFunctor<double>);
```

Comment:
Perfect. :+1:

Reply:
So here, as you mentioned yesterday, the last two arguments of `milu0_decomposition` are functions that receive double and return `const double& `type objects. So the (only) the name of those functions(with the type double since they were defined in a `template <typename T>`) should be the argument, **without the extra ()**.
Thanks again!